telegram/bot.py

```diff
@@ -728,6 +743,47 @@ def sendContact(self, chat_id, phone_number, first_name, last_name=None, **kwarg
 
     @log
     @message
+    def sendGame(self,
+                 chat_id,
+                 game_short_name,
+                 parse_mode=None,
+                 disable_web_page_preview=None,
```

Comment:
According to https://core.telegram.org/bots/api#sendgame the kwargs should be different